dts/bindings/display/zephyr,dummy-dc.yaml

```diff
@@ -6,3 +6,18 @@ description: Dummy display controller
 compatible: "zephyr,dummy-dc"
 
 include: display-controller.yaml
+
+properties:
+  pixel-format:
+    type: int
+    default: 1
+    enum:
+      - 1  # PIXEL_FORMAT_RGB_888
+      - 2  # PIXEL_FORMAT_MONO01
+      - 4  # PIXEL_FORMAT_MONO10
+      - 8  # PIXEL_FORMAT_ARGB_8888
+      - 16 # PIXEL_FORMAT_RGB_565
+      - 32 # PIXEL_FORMAT_BGR_565
```

Comment:
Maybe more descriptive would be to use a string. You could use `DT_INST_STRING_UPPER_TOKEN` to map directly to the values (add a common prefix).

Reply:
Have not used this as of yet, you mean that I take a string uppercased from DT, store it in the data and use a lookup map in the `init` to set the format?

Checked other displays like `ilitek,ili9xxx-common.yaml` and they are using the same. Maybe adding a dts include header with the values would be a good comprimise?

Reply:
Forgot to ask, but do you consider the panel.h solution as good enough or do you want to see the stringification approach instead?

Reply:
Re-using `panel.h` is good enough, I think.